docs/core/testing/unit-testing-platform-intro.md

```diff
@@ -12,6 +12,10 @@ Microsoft.Testing.Platform is a lightweight and portable alternative to [VSTest]
 
 `Microsoft.Testing.Platform` is open source. You can find `Microsoft.Testing.Platform` code in [microsoft/testfx](https://github.com/microsoft/testfx/tree/main/src/Platform/Microsoft.Testing.Platform) GitHub repository.
 
+# Supported test frameworks
+
+* MSTest a.k.a. [MSTest runner](unit-testing-mstest-runner-intro.md)
```

Comment:
I think the section is good. I would maybe changed the wording for MSTest runner.

Something like

MSTest. In MSTest, the support of Microsoft.Testing.Platform is done via MSTest runner.

(I'm on phone so I have kept the back quote or links)

Reply:
applied